src/client/partition.rs

```diff
@@ -399,7 +399,6 @@ fn build_produce_request(
 
     // TODO: Retry on failure
 
-    // TODO: Verify this is the first timestamp in the batch and not the min
```

Comment:
I'm guessing the conclusion is it is the first and not the min?

Reply:
yes, ensured by the cross-tests with the other implementations.